sdk/core/System.Net.ClientModel/src/ModelReaderWriter/ModelReaderWriter.cs

```diff
@@ -12,6 +13,21 @@ namespace System.Net.ClientModel.Core
     /// </summary>
     public static class ModelReaderWriter
     {
+        /// <summary>
+        /// Creates an instance of a model from an instance of an anonymous type.
+        /// </summary>
+        /// <typeparam name="T">The type of the model to create.</typeparam>
+        /// <param name="jsonSerializable">An instance of an anonymous type representing the model</param>
+        /// <returns>A <typeparamref name="T"/> instance.</returns>
+        public static T? Create<T>(object jsonSerializable) where T : IJsonModel<T>
+        {
+            BinaryData json = BinaryData.FromObjectAsJson(jsonSerializable);
+            var reader = new Utf8JsonReader(json);
```

Comment:
nit: pls don't use `var`

Reply:
I thought our guidance is that we don't use var in cases where it's not clear what the type of the variable would be. When newing up a new instance, it's very clear. In fact, I think it's less readable to not use var in such cases.

Reply:
That is fair.  I guess it is just my personal preference because it's easier for me to understand the code if the variable type declarations are lined up on the left.

But I don't feel super strongly, so if you do, I'm fine with letting it go and moving forward.

Reply:
@heaths, do you know whether we have an official or semi-official convention on our team on whether to use e.g. var x = new X() or X x = new()?

Reply:
`var` is acceptable when obvious. The root `.editorconfig` should have some of that codified. VS will use it by default, and you can install the EditorConfig.EditorConfig extension for VSCode. There are plugins for other editors, too.

Reply:
That's cool, I'll let Krzysztof use `var` here and take myself to go sit in the corner and tap the space bar in multiples of four into my emacs editor.  😉  <Walks off in her **"SPACES; EMACS; X x = new()"** t-shirt/>

In all seriousness, though, happy to align with the team on this one and let us all get back to work 🙂